### ISSUE TITLE
Powershell Core profile

### ISSUE THREAD

**Issue by Ash258** [lukesampson/pshazz#73]
Pshazz is not initialized properly when used inside pwsh.

PWSH and normal powershell use different profile locations. (https://github.com/PowerShell/PowerShell/issues/4682) 

Creating symbolic link for profile file is working.

PWSH:
```powershell
$ $PROFILE
C:\Users\Ash258\Documents\PowerShell\Microsoft.PowerShell_profile.ps1
```

Powershell:
```powershell
$ $PROFILE
C:\Users\Ash258\Documents\WindowsPowerShell\Microsoft.PowerShell_profile.ps1
```

![Screen](https://i.imgur.com/FCXzyuO.png)
Already created symlink for profile.

**Comment by qcts33** [comment]
The install script of pshazz seems alright, so a another quick walk around is just uninstall and reinstall pshazz inside the pwsh

**Comment by cderv** [comment]
> The install script of pshazz seems alright, so a another quick walk around is just uninstall and reinstall pshazz inside the pwsh

Yes that worked for me. After uninstalling and reinstalling inside pwsh, the profile is installed and then loaded correctly.
